addons/product/product_view.xml

```diff
@@ -53,13 +53,14 @@
             <field name="model">product.template</field>
             <field name="arch" type="xml">
                 <tree string="Product">
+                    <field name="default_code"/>
                     <field name="sequence" widget="handle"/>
                     <field name="name"/>
                     <field name="list_price"/>
                     <field name="standard_price"/>
                     <field name="categ_id"/>
                     <field name="type"/>
-                    <field name="uom_id" invisible="1"/>
+                    <field name="uom_id" options="{'no_open': True, 'no_create': True}" groups="product.group_uom"/>
```

Comment:
Please set the unit of measure after the quantity on hand and the forecast quantity (like on the product variants).